package.json

```diff
@@ -39,7 +39,7 @@
     "pub": "npm run compile && npm run build && npm publish"
   },
   "peerDependencies": {
-    "g2": "^1.2.6"
+    "g2": "^2.0.0"
```

Comment:
this will cause install error when using g2@1.x with g2-react, why use word of fix?